airbyte-cdk/python/unit_tests/sources/file_based/scenarios/validation_policy_scenarios.py

```diff
@@ -12,24 +12,24 @@
             "a.csv": {  # The records in this file do not conform to the schema
                 "contents": [
                     ("col1", "col2"),
-                    ("val_a_11", "val_a_21"),
-                    ("val_a_12", "val_a_22"),
+                    ("val_a_11", "1"),
+                    ("val_a_12", "2"),
                 ],
                 "last_modified": "2023-06-05T03:54:07.000Z",
             },
             "b.csv": {
                 "contents": [
-                    ("col1",),
-                    ("val_b_11",),
-                    ("val_b_12",),
+                    ("col1", "col2"),
```

Comment:
We started to consider `val_c_12,val_c_22` as a parsing error since there is more columns than headers and this is not a valid csv file. This leads to two things:
* the cdk stop to process this file and therefore entry `val_c_13` will not be emitted
* there are no validation policies being triggered anymore in this dataset

If we do not consider this a parsing error, we can revert this file. Note that we also consider having too few columns as a parsing error so we would only emit the first row of the following CSV:

```
header1,header2
1,2
11,22,33
111,222
```

Reply:
that makes sense to me. thanks!